api/server/router/network/network_routes.go

```diff
@@ -213,6 +213,7 @@ func buildIpamResources(r *types.NetworkResource, nw libnetwork.Network) {
 		iData.IPRange = ip4.SubPool
 		iData.Gateway = ip4.Gateway
 		iData.AuxAddress = ip4.AuxAddresses
+		iData.Options = ip4.Options
```

Comment:
above comment is applicable here as well. i dont think `options` belong to this structure.